tests/playwright/sanity/assets/js/frontend/utils/url-actions.test.js

```diff
@@ -0,0 +1,184 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../../../../../pages/wp-admin-page' );
+
+test.describe( 'URL Actions', () => {
+	test.only( 'Test Lightbox and URL Actions', async ( { page }, testInfo ) => {
+		/**
+		 * Open new empty page.
+		 */
+		const wpAdmin = new WpAdminPage( page, testInfo );
+
+		const editor = await wpAdmin.useElementorCleanPost();
+
+		const wpMediaAddButtonSelector = '.button.media-button';
+
+		/**
+		 * Add Image Widget and test lightbox in single image mode.
+		 */
+		await editor.addElement( {
+			elType: 'widget',
+			widgetType: 'image',
+			settings: {
+				link_to: 'file',
+				open_lightbox: 'yes',
+			},
+		}, null );
+
+		// Upload Image
+		const imageMediaControl = await page.locator( '.elementor-control-preview-area' );
+		await imageMediaControl.click( { delay: 500, clickCount: 2 } );
+		await page.click( 'text=Media Library' );
+		await page.waitForSelector( 'text=Insert Media' );
+		// takes time to load the media library assets.
+		await page.waitForTimeout( 2000 );
```

Comment:
Please try to replace all timeouts with `waitForSelector`

Reply:
Done where it was possible